recipe/meta.yaml

```diff
@@ -152,6 +154,88 @@ outputs:
       license: MIT
       license_file: LICENSE.TXT
 
+  - name: libgdal-arrow-parquet
+    script: build_arrow_parquet.sh  # [unix]
+    script: build_arrow_parquet.bat  # [win]
+    requirements:
+      build:
+        - cmake
+        # ranlib used at install time
+        - {{ compiler('c') }}
+        # libstdc++ is needed in requirements/run
+        - {{ compiler('cxx') }}
+        - pkg-config  # [not win]
+        - make  # [unix]
+        - ninja  # [win]
+        - sysroot_linux-64 2.17  # [linux64]
+      host:
+        - {{ pin_subpackage('libgdal', exact=True) }}
+        - arrow-cpp >=10.0
+        - blosc
+        - cfitsio
+        - curl
+        - expat
+        - freexl
+        - geos
+        - geotiff
+        - giflib  # [not win]
+        - hdf4
+        - hdf5
+        - icu
+        - jpeg
+        - json-c  # [not win]
+        - kealib
+        - libkml
+        - libnetcdf
+        - libpng
+        - libpq
+        - libspatialite
+        - libtiff
+        - libuuid  # [linux]
+        - libwebp-base
+        - libxml2
+        - openjpeg
+        - openssl
+        - pcre2
+        - poppler
+        - postgresql
+        - proj
+        - qhull
+        - sqlite
+        - tiledb
+        - xerces-c
+        - xz
+        - zlib
+        - zstd
+      run:
+        - {{ pin_subpackage('libgdal', exact=True) }}
+        - arrow-cpp
```

Comment:
Unless you were relying on the python bits in arrow-cpp, you don't need this anymore; libarrow has a run-export.
```suggestion
```